bucket/chromium.json

```diff
@@ -31,21 +31,22 @@
     ],
     "checkver": {
         "github": "https://github.com/macchrome/winchrome",
-        "regex": "v([\\d.]+-r\\d+)-Win(64|32)"
+        "regex": "/v(?<version>[\\d.]+)-r(?<build>[\\d]+)-Win64/ungoogled-chromium-(?<version>[\\d.]+)-1_Win(64|32)\\.7z",
+        "replace": "${version}"
```

Comment:
Let the final version string be 95.0.4638.69-r920003-1 for this one.

Safer. 

@rashil2000 @issaclin32 your views?

Reply:
I agree. This will help us to keep track of updates (and hotfixes) better.